kahuna/public/js/search/query.html

```diff
@@ -62,7 +62,14 @@
                                gu:preset-dates="searchQuery.sinceOptions"
                                gu:first-day="1">
                 </gu-date-range>
-            <li>
+            </li>
+
+            <li class="search__filter-item">
+                <select class="search__filter"
+                        ng:model="searchQuery.filter.orderBy"
+                        ng:options="sortOrder.value as sortOrder.label for sortOrder in searchQuery.orderByOptions">
+                </select>
+            </li>
```

Comment:
Re saving space, Drive's solution:

![screen shot 2015-07-28 at 11 21 42](https://cloud.githubusercontent.com/assets/836140/8929130/27abef4c-351b-11e5-9754-d35772c2e06a.png)


Not recommending, but its interesting. I like that its size is small (just an icon) but I don't like that its not immediately obvious what you've got selected - you have to open the dropdown and see what's ticked.